bundler/src/main.rs

```diff
@@ -204,6 +205,13 @@ async fn bundle_endpoint(
     }
 }
 
+/// Returns an HTTP 200 response when requested.
+///
+/// Failures in the bundle update and serialization result in service crash, so ability to serve this endpoint implies liveness
+async fn health_endpoint() -> impl IntoResponse {
+    StatusCode::OK
```

Comment:
Should readiness wait until there is a bundle that is built and servable?

Reply:
The API doesn't get served until the initial bundle has been fetched & built, so it indirectly does